### PR TITLE
115 fortunes algorithm

### LINKED ISSUE #115: Fortune's algorithm

### PR DESCRIPTION
**Pull request by VictorVader**

### CODE REVIEW

File: backend/cmd/k-means/kMeans_test.go [hunk 1]
```diff
@@ -16,87 +16,87 @@ import (
...
-			// Check that each cluster has at least one observation
-			for _, c := range got {
-				if len(c.ZoneRanges) == 0 {
-					t.Errorf("kMeans() returned a cluster with 0 observations")
-				}
-			}
-		})
-	}
-}
+// func TestKMeans(t *testing.T) {
```
Comment (PhilipWaldman) [inline backend/cmd/k-means/kMeans_test.go:20]: If you're going to comment this whole function out, you might as well remove it completely.

Comment (Lucasmeteenc) [reply backend/cmd/k-means/kMeans_test.go:20]: @VictorVader can you add //TODO:
@PhilipWaldman This test is necessary to test KMeans, but due to the change in handlers I had to change the ouput of the algo and didn't have time to fix the test

**Review by Lucasmeteenc — DISMISSED** [review]
Backend is fine, didn't look at the frontend

**Review by PhilipWaldman — DISMISSED** [review]
Backend is fine. The frontend looked fine to me too, but someone from frontend should check it just to make sure

File: backend/cmd/k-means/kMeans_test.go (hunk 1 above)
Comment (VictorVader) [reply backend/cmd/k-means/kMeans_test.go:20]: Added, you'll need to re approve

**Review by PhilipWaldman — APPROVED** [review]

**Review by Patricijia — APPROVED** [review]

**Comment by andreacbeiu** [comment]
Front-end approved! :))

**Pull request merged**